base/float.jl

```diff
@@ -711,6 +711,69 @@ end
     eps() = eps(Float64)
 end
 
+"""
+    eps{T<:AbstractFloat}(::Type{T})
+    eps()
+
+Returns the *machine epsilon* of the floating point type `T` (`T = Float64` by
+default). This is defined as the gap between 1 and the next largest value representable by
+`T`, and is equivalent to `eps(one(T))`.
+
+```jldoctest
+julia> eps()
+2.220446049250313e-16
+
+julia> eps(Float32)
+1.1920929f-7
+
+julia> 1.0 + eps()
+1.0000000000000002
+
+julia> 1.0 + eps()/2
+1.0
+```
+"""
+eps{T<:AbstractFloat}(::Type{T})
+
+"""
+    eps(x::AbstractFloat)
+
+Returns the *unit in last place* (ulp) of `x`. This is the distance between consecutive
+representable floating point values at `x` of the same type as `x`. In most cases, if the
+distance on either side of `x` is different then the larger of the two is taken, that is
+
+    eps(x) == max(x-prevfloat(x), nextfloat(x)-x)
+
+with the exception of the smallest and largest finite values (`nextfloat(-Inf)` and
+`prevfloat(Inf)`), which round to the smaller of the values.
+
+The rationale for this choice is that under default `RoundNearest` behaviour, for any
```

Comment:
If I'm not mistaken, the docs mostly use American English, so I think "behaviour" should be "behavior."